tests/core/test_run.py

```diff
@@ -22,14 +22,14 @@ def _expected_runs_table(*row_default):
 @all_platforms
 class TestCoreRun:
     def test_run_versions(self, test_mp):
-        run1 = test_mp.Run("Model", "Scenario", version="new")
```

Comment:
We should probably add a test for the `Run.NotFound` exception here now?

Reply:
Thanks, added a test